csi.proto

```diff
@@ -131,6 +131,14 @@ message PluginCapability {
       // attempt to invoke the REQUIRED ControllerService RPCs, as well
       // as specific RPCs as indicated by ControllerGetCapabilities.
       CONTROLLER_SERVICE = 1;
+
+      // ACCESSIBILITY_CONSTRAINTS indicates that the volumes for this
+      // plugin may not be equally accessible by all nodes in the
+      // cluster. The CO SHOULD use the topology information returned by
+      // CreateVolumeRequest along with the topology information
+      // returned by NodeGetId to ensure that a given volume is
+      // accessible from a given node when scheduling workloads.
```

Comment:
Would it not be better to say "The CO MUST use the topology information..." since failure to do so could cause workloads to crash?

Reply:
Good point, will change.